dist/moduleResults/moduleResults.js

```diff
@@ -0,0 +1,79 @@
+(function (angular) {
+    'use strict';
+    angular.module('znk.infra.moduleResults', []);
+})(angular);
+
+(function (angular) {
+    'use strict';
+
+    angular.module('znk.infra.moduleResults').service('ModuleResultsService', [
+        'InfraConfigSrv', '$log', '$q', 'UtilitySrv',
+        function (InfraConfigSrv, $log, $q, UtilitySrv) {
+
+            var moduleResultsService = {};
+            var storage = InfraConfigSrv.getStorageService();
+            var USER_MODULE_RESULTS_PATH = storage.variables.appUserSpacePath + '/moduleResults';
+            var MODULE_RESULTS_PATH = 'moduleResults';
+
+            moduleResultsService.getUserModuleResultsGuids = function (userId){
+                var userResultsPath = USER_MODULE_RESULTS_PATH.replace('$$uid', userId);
+                return storage.get(userResultsPath);
+            };
+
+            moduleResultsService.getModuleResultByGuid = function (resultGuid, defaultValue) {
+                var resultPath = MODULE_RESULTS_PATH + '/' + resultGuid;
+                return storage.get(resultPath, defaultValue);
+            };
+
+            moduleResultsService.getModuleResultById = function (moduleId, userId, withDefaultResult) {
```

Comment:
1. the function return result for specific module id. I canged the name to getModuleResultByModuleId
2. getModuleResultByGuid used for request getUserModuleResultsGuids (list of guids), then I used getModuleResultByGuid to get specific results.

getModuleResultByModuleId will be used in the student app